analytics/terraform/spark-k8s-operator/examples/karpenter/spark-mountpoint-s3/blueprints/spark-mountpoint-s3-3.4.0.yaml

```diff
@@ -0,0 +1,132 @@
+# Pre-requisite before running this job
+# Replace <S3_BUCKET_NAME> with your S3 bucket created by this blueprint(Check Terraform outputs)
+# Verify that the version for spark matches 
+
+---
+apiVersion: "sparkoperator.k8s.io/v1beta2"
+kind: SparkApplication
+metadata:
+  name: "taxi-trip"
+  namespace: spark-team-a
+  labels:
+    app: "taxi-trip"
+    applicationId: "taxi-trip-nvme"
+    # Assign the job to a Yunikorn Queue via label.
+    queue: root.test
+spec:
+#  To create Ingress object for Spark driver.
+#  Ensure Spark Operator Helm Chart deployed with Ingress enabled to use this feature
+#  sparkUIOptions:
+#    servicePort: 4040
+#    servicePortName: taxi-trip-ui-svc
+#    serviceType: ClusterIP
+#    ingressAnnotations:
+#      kubernetes.io/ingress.class: nginx
+#      nginx.ingress.kubernetes.io/use-regex: "true"
```

Comment:
Remove the commented lines and keep the yaml simple and highlight the feature config lines by adding comments